frontend/src/pages/pipelines/global/experiments/useDefaultExperiment.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { usePipelinesAPI } from '~/concepts/pipelines/context';
+import { ExperimentKFv2 } from '~/concepts/pipelines/kfTypes';
+import useFetchState, { FetchState, FetchStateCallbackPromise } from '~/utilities/useFetchState';
+
+/**
+ * Fetch the first created experiment and check it's name to make sure it's default experiment
+ */
+const useDefaultExperiment = (): FetchState<ExperimentKFv2 | null> => {
+  const { api } = usePipelinesAPI();
+
+  const getDefaultExperiment = React.useCallback<
+    FetchStateCallbackPromise<ExperimentKFv2 | null>
+  >(async () => {
+    const response = await api.listExperiments(
+      {},
+      {
+        sortField: 'created_at',
+        sortDirection: 'asc',
+        pageSize: 1,
+      },
+    );
+
+    return response.experiments?.[0].display_name === 'Default' ? response.experiments[0] : null;
```

Comment:
I question using the name and not just the first created, as I'm not sure if the name is subject to change in the API. I would double check with them to at least make sure they have no plans on changing the name going forward.

Reply:
After discussion here https://redhat-internal.slack.com/archives/C06SMDJGZC2/p1721400438442669. I will just update this search with the name `Default`.